frontend/styles/var/mixins.scss

```diff
@@ -241,14 +245,30 @@ $max-width: 2560px;
   }
 }
 
-@mixin flex-column-1rem-gap {
+@mixin flex-column-16px-gap {
   display: flex;
   flex-direction: column;
-  gap: $pad-medium;
+  gap: $gap-form;
+}
+
+@mixin flex-column-24px-gap {
+  display: flex;
+  flex-direction: column;
+  gap: $gap-page-component;
 }
 
 @mixin page {
-  @include flex-column-1rem-gap;
+  @include flex-column-24px-gap;
+}
+
+// Separate in case there's any card level styling we need
+@mixin card {
+  @include flex-column-24px-gap;
+}
```

Comment:
im a bit confused about this mixin. I can see its being used to override card styles in AuthenticationFormWrapper and nowhere else. can we just include the overrides in that component?

Reply:
It's being used in 13 cards in the final revamp found on the everything branch `ui-reskin`

<img width="862" height="908" alt="Screenshot 2025-09-19 at 12 39 41 PM" src="https://github.com/user-attachments/assets/c1d9f24b-d058-4bfa-8717-1ad6bb25a467" />